src/app/child-dev-project/attendance/attendance.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from "@angular/core";
+import { EntityMapperService } from "../../core/entity/entity-mapper.service";
+import moment from "moment";
+import { Note } from "../notes/model/note";
+
+@Injectable({
+  providedIn: "root",
+})
+export class AttendanceService {
+  constructor(private entityMapper: EntityMapperService) {}
+
+  async getEventsOnDate(date: Date) {
+    const events = await this.entityMapper.loadType<Note>(Note);
```

Comment:
This could definitely be speed up by a index which is sorted by the event date.

Reply:
that's on the todo list but maybe deserves its on PR